--- ./src/common/CMakeLists.txt	2026-03-18 12:59:19.778496343 +0100
+++ ./src/common/CMakeLists.txt	2026-03-16 11:44:06.236789074 +0100
@@ -185,6 +185,11 @@
 set(TARGET_TYPE SHARED)
 add_library(meshlab-common ${TARGET_TYPE} ${SOURCES} ${HEADERS} ${RESOURCES})
 
+if(NOT WIN32 AND NOT APPLE)
+    target_compile_definitions(
+        meshlab-common PRIVATE MESHLAB_LIB_INSTALL_DIR="${MESHLAB_LIB_INSTALL_DIR}")
+endif()
+
 target_compile_definitions(meshlab-common
 	PUBLIC
 		MESHLAB_VERSION=${MESHLAB_VERSION}
--- ./src/common/globals.cpp	2026-03-18 12:58:56.870312860 +0100
+++ ./src/common/globals.cpp	2026-03-16 11:44:06.236789074 +0100
@@ -36,6 +36,20 @@
 {
 	QDir baseDir(qApp->applicationDirPath());
 
+#if defined(Q_OS_LINUX)
+	if (baseDir.dirName() == "bin") {
+		baseDir.cdUp();
+#ifdef MESHLAB_LIB_INSTALL_DIR
+		baseDir.cd(MESHLAB_LIB_INSTALL_DIR);
+#else
+		baseDir.cd("lib");
+		if(baseDir.exists("meshlab")) {
+			baseDir.cd("meshlab");
+		}
+#endif // MESHLAB_LIB_INSTALL_DIR
+	}
+#endif
+
 #ifdef Q_OS_WIN
 	// Windows:
 	// during development with visual studio binary could be in the debug/release subdir.
