diff -Naur a/setup.py b/setup.py
--- a/setup.py	2020-12-04 02:52:19.000000000 +0100
+++ b/setup.py	2021-01-04 13:15:49.785801259 +0100
@@ -78,7 +78,8 @@
 def find_library(name, vision_include):
     this_dir = os.path.dirname(os.path.abspath(__file__))
     build_prefix = os.environ.get('BUILD_PREFIX', None)
-    is_conda_build = build_prefix is not None
+#    is_conda_build = build_prefix is not None
+    is_conda_build = False
 
     library_found = False
     conda_installed = False
