--- a/configure.ac	2016-04-05 10:09:01.000000000 +0000
+++ b/configure.ac	2018-01-07 04:27:49.195251198 +0000
@@ -265,7 +265,12 @@
 CPPFLAGS="$old_CPPFLAGS"
 LIBS="$old_LIBS"
 
-PKG_CHECK_MODULES([SYSTEMD], [libsystemd >= 209], [USE_SD_NOTIFY=1], [USE_SD_NOTIFY=0])
+AC_ARG_ENABLE([systemd], AS_HELP_STRING([--enable-systemd], [Enable systemd notify support (default: disabled)]))
+AS_IF([test "x$enable_systemd" = "xyes"], [
+	PKG_CHECK_MODULES([SYSTEMD], [libsystemd >= 209], [USE_SD_NOTIFY=1], [USE_SD_NOTIFY=0])
+], [
+	USE_SD_NOTIFY=0
+])
 AC_SUBST(USE_SD_NOTIFY)
 
 AC_CHECK_LIB(resolv, res_gethostbyname, RESOLV="-lresolv", RESOLV="")
