From 7c4c9a6d0977a75ddb768ad0a3087d033d3bc9e6 Mon Sep 17 00:00:00 2001
From: Will <22566733+willh-cmyk@users.noreply.github.com>
Date: Thu, 11 Apr 2024 11:18:45 +0100
Subject: [PATCH] specify wheel build target explicitly

---
 pyproject.toml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/pyproject.toml b/pyproject.toml
index 4c7bc717..d951dec0 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -62,6 +62,11 @@ artifacts = [
 # include is required since the project name doesn't match the folder name
 include = ["kubespawner"]
 
+[tool.hatch.build.targets.wheel]
+# packages is required when building wheels since the project name doesn't match
+# the folder name.
+packages = ["kubespawner"]
+
 # black is used for autoformatting Python code
 #
 # ref: https://black.readthedocs.io/en/stable/
