diff -Naur a/api/contrib/quotas.py b/api/contrib/quotas.py
--- a/cinder/api/contrib/quotas.py	2016-03-01 13:52:39.671576074 +0100
+++ b/cinder/api/contrib/quotas.py	2016-03-01 13:53:19.073740023 +0100
@@ -186,7 +186,8 @@
                 auth_url=CONF.keystone_authtoken.auth_uri,
                 token=context.auth_token,
                 project_id=context.project_id)
-            client_session = session.Session(auth=auth_plugin)
+            verify = not CONF.keystone_authtoken.insecure
+            client_session = session.Session(auth=auth_plugin, verify=verify)
             keystone = client.Client(auth_url=CONF.keystone_authtoken.auth_uri,
                                      session=client_session)
             project = keystone.projects.get(id, subtree_as_ids=subtree_as_ids)
