diff -Naur a/nvidiaOpticalFlow.cpp b/nvidiaOpticalFlow.cpp
--- a/opencv_contrib-4.1.2/modules/cudaoptflow/src/nvidiaOpticalFlow.cpp	2020-05-13 14:32:45.608691285 +0200
+++ b/opencv_contrib-4.1.2/modules/cudaoptflow/src/nvidiaOpticalFlow.cpp	2020-05-13 14:37:19.256704334 +0200
@@ -8,11 +8,14 @@
 
 #if !defined HAVE_CUDA || defined(CUDA_DISABLER)
 
-Ptr<NvidiaOpticalFlow_1_0> cv::cuda::NvidiaOpticalFlow_1_0::create(int, int, int, NVIDIA_OF_PERF_LEVEL, bool, bool) { throw_no_cuda(); return Ptr<NvidiaOpticalFlow_1_0>(); }
+cv::Ptr<cv::cuda::NvidiaOpticalFlow_1_0> cv::cuda::NvidiaOpticalFlow_1_0::create(int, int, NVIDIA_OF_PERF_LEVEL, bool, bool, bool, int) { throw_no_cuda(); return cv::Ptr<cv::cuda::NvidiaOpticalFlow_1_0>(); }
 
 #elif !defined HAVE_NVIDIA_OPTFLOW
 
-CV_Error(cv::Error::HeaderIsNull, "Nvidia Optical Flow headers not found. Make sure cmake downloads it properly");
+cv::Ptr<cv::cuda::NvidiaOpticalFlow_1_0> cv::cuda::NvidiaOpticalFlow_1_0::create(int, int, NVIDIA_OF_PERF_LEVEL, bool, bool, bool, int)
+{
+    CV_Error(cv::Error::HeaderIsNull, "OpenCV was build without NVIDIA OpticalFlow support");
+}
 
 #else
 
@@ -653,4 +656,4 @@
         bEnableCostBuffer,
         gpuId);
 }
-#endif
\ No newline at end of file
+#endif
