--- opencv-4.1.2_org/cmake/OpenCVDetectPython.cmake	2019-10-10 00:53:14.000000000 +0200
+++ opencv-4.1.2/cmake/OpenCVDetectPython.cmake	2020-05-13 13:29:17.259725265 +0200
@@ -186,7 +186,11 @@
         else() #debian based assumed, install to the dist-packages.
           set(_packages_path "python${_version_major_minor}/dist-packages")
         endif()
-        set(_packages_path "lib/${_packages_path}")
+	if("${_std_packages_path}" MATCHES "lib64")
+          set(_packages_path "lib64/${_packages_path}")
+	else()
+          set(_packages_path "lib/${_packages_path}")
+	endif()
       elseif(CMAKE_HOST_WIN32)
         get_filename_component(_path "${_executable}" PATH)
         file(TO_CMAKE_PATH "${_path}" _path)
