diff -Naur ./__init__.py ./__init__.py
--- a/__init__.py	2024-04-29 23:23:38.715035555 -0000
+++ b/__init__.py	2024-04-29 23:23:08.723037884 -0000
@@ -109,8 +109,8 @@
         from .bgprocess import blueprint as module
         self.submodules.append(module)
 
-        from .cloud import blueprint as module
-        self.submodules.append(module)
+#        from .cloud import blueprint as module
+#        self.submodules.append(module)
 
         from .dependencies import blueprint as module
         self.submodules.append(module)
