diff -Naur isis/src/base/apps/ckwriter/SpiceSegment.h ../isis3_cmake_working/isis/src/base/apps/ckwriter/SpiceSegment.h
--- isis/src/base/apps/ckwriter/SpiceSegment.h	2018-04-06 15:12:14.724517336 +0200
+++ ./isis3_cmake_working/isis/src/base/apps/ckwriter/SpiceSegment.h	2018-04-03 13:26:47.739150275 +0200
@@ -36,10 +36,10 @@
 
 #include <SpiceZdf.h>
 
-#include <tnt/tnt_array1d.h>
-#include <tnt/tnt_array1d_utils.h>
-#include <tnt/tnt_array2d.h>
-#include <tnt/tnt_array2d_utils.h>
+#include <tnt_array1d.h>
+#include <tnt_array1d_utils.h>
+#include <tnt_array2d.h>
+#include <tnt_array2d_utils.h>
 
 #include "Kernels.h"
 #include "IException.h"
diff -Naur isis/src/base/apps/equalizer/equalizer.cpp ../isis3_cmake_working/isis/src/base/apps/equalizer/equalizer.cpp
--- isis/src/base/apps/equalizer/equalizer.cpp	2018-04-06 15:12:14.768517608 +0200
+++ ./isis3_cmake_working/isis/src/base/apps/equalizer/equalizer.cpp	2018-04-03 13:26:47.530148951 +0200
@@ -2,9 +2,9 @@
 
 #include <cmath>
 
-#include <jama/jama_svd.h>
+#include <jama_svd.h>
 #include <QString>
-#include <tnt/tnt_array2d.h>
+#include <tnt_array2d.h>
 
 #include "CubeAttribute.h"
 #include "Equalization.h"
diff -Naur isis/src/base/apps/spkwriter/SpiceSegment.h ../isis3_cmake_working/isis/src/base/apps/spkwriter/SpiceSegment.h
--- isis/src/base/apps/spkwriter/SpiceSegment.h	2018-04-06 15:12:14.999519040 +0200
+++ ./isis3_cmake_working/isis/src/base/apps/spkwriter/SpiceSegment.h	2018-04-03 13:26:47.942151561 +0200
@@ -33,10 +33,10 @@
 
 #include <SpiceZdf.h>
 
-#include <tnt/tnt_array1d.h>
-#include <tnt/tnt_array1d_utils.h>
-#include <tnt/tnt_array2d.h>
-#include <tnt/tnt_array2d_utils.h>
+#include <tnt_array1d.h>
+#include <tnt_array1d_utils.h>
+#include <tnt_array2d.h>
+#include <tnt_array2d_utils.h>
 
 #include "Cube.h"
 #include "IException.h"
diff -Naur isis/src/base/objs/Affine/Affine.cpp ../isis3_cmake_working/isis/src/base/objs/Affine/Affine.cpp
--- isis/src/base/objs/Affine/Affine.cpp	2018-04-06 15:12:15.026519207 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/Affine/Affine.cpp	2018-04-03 13:26:47.045145880 +0200
@@ -26,7 +26,7 @@
 #include <sstream>
 #include <string>
 
-#include <jama/jama_svd.h>
+#include <jama_svd.h>
 
 #include "Constants.h"
 #include "IException.h"
diff -Naur isis/src/base/objs/Affine/Affine.h ../isis3_cmake_working/isis/src/base/objs/Affine/Affine.h
--- isis/src/base/objs/Affine/Affine.h	2018-04-06 15:12:15.026519207 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/Affine/Affine.h	2018-04-03 13:26:47.044145873 +0200
@@ -23,7 +23,7 @@
  */
 
 #include <vector>
-#include "tnt/tnt_array2d.h"
+#include "tnt_array2d.h"
 
 namespace Isis {
   /**
diff -Naur isis/src/base/objs/Affine/unitTest.cpp ../isis3_cmake_working/isis/src/base/objs/Affine/unitTest.cpp
--- isis/src/base/objs/Affine/unitTest.cpp	2018-04-06 15:12:15.026519207 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/Affine/unitTest.cpp	2018-04-03 13:26:47.044145873 +0200
@@ -3,7 +3,7 @@
 
 #include "Affine.h"
 #include "Preference.h"
-#include "tnt/tnt_array2d_utils.h"
+#include "tnt_array2d_utils.h"
 
 using namespace std;
 using namespace TNT;
diff -Naur isis/src/base/objs/Blobber/Blobber.h ../isis3_cmake_working/isis/src/base/objs/Blobber/Blobber.h
--- isis/src/base/objs/Blobber/Blobber.h	2018-04-06 15:12:15.032519245 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/Blobber/Blobber.h	2018-04-03 13:26:46.913145044 +0200
@@ -25,7 +25,7 @@
 
 #include <QString>
 
-#include <tnt/tnt_array2d.h>
+#include <tnt_array2d.h>
 
 namespace Isis {
   class Cube;
diff -Naur isis/src/base/objs/CSVReader/CSVReader.h ../isis3_cmake_working/isis/src/base/objs/CSVReader/CSVReader.h
--- isis/src/base/objs/CSVReader/CSVReader.h	2018-04-06 15:12:15.034519257 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/CSVReader/CSVReader.h	2018-04-03 13:26:47.037145829 +0200
@@ -26,7 +26,7 @@
 #include <QString>
 #include <vector>
 #include <fstream>
-#include <tnt/tnt_array1d.h>
+#include <tnt_array1d.h>
 #include "CollectorMap.h"
 #include "IException.h"
 #include "IString.h"
diff -Naur isis/src/base/objs/Chip/Chip.cpp ../isis3_cmake_working/isis/src/base/objs/Chip/Chip.cpp
--- isis/src/base/objs/Chip/Chip.cpp	2018-04-06 15:12:15.038519282 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/Chip/Chip.cpp	2018-04-03 13:26:47.000145594 +0200
@@ -44,7 +44,7 @@
 #include <vector>
 
 #include <geos/geom/Point.h>
-#include <tnt/tnt_array2d_utils.h>
+#include <tnt_array2d_utils.h>
 
 
 
diff -Naur isis/src/base/objs/ForstnerOperator/ForstnerOperator.cpp ../isis3_cmake_working/isis/src/base/objs/ForstnerOperator/ForstnerOperator.cpp
--- isis/src/base/objs/ForstnerOperator/ForstnerOperator.cpp	2018-04-06 15:12:15.053519375 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/ForstnerOperator/ForstnerOperator.cpp	2018-04-03 13:26:46.956145316 +0200
@@ -1,8 +1,8 @@
 #include "ForstnerOperator.h"
 #include "Chip.h"
 #include "FourierTransform.h"
-#include "tnt/tnt_array2d.h"
-#include "jama/jama_lu.h"
+#include "tnt_array2d.h"
+#include "jama_lu.h"
 #include <complex>
 
 namespace Isis {
diff -Naur isis/src/base/objs/GSLUtility/GSLUtility.h ../isis3_cmake_working/isis/src/base/objs/GSLUtility/GSLUtility.h
--- isis/src/base/objs/GSLUtility/GSLUtility.h	2018-04-06 15:12:15.054519381 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/GSLUtility/GSLUtility.h	2018-04-03 13:26:46.882144847 +0200
@@ -32,10 +32,10 @@
 
 #include <QString>
 
-#include <tnt/tnt_array1d.h>
-#include <tnt/tnt_array1d_utils.h>
-#include <tnt/tnt_array2d.h>
-#include <tnt/tnt_array2d_utils.h>
+#include <tnt_array1d.h>
+#include <tnt_array1d_utils.h>
+#include <tnt_array2d.h>
+#include <tnt_array2d_utils.h>
 
 #include <gsl/gsl_vector.h>
 #include <gsl/gsl_matrix.h>
diff -Naur isis/src/base/objs/Gruen/Gruen.cpp ../isis3_cmake_working/isis/src/base/objs/Gruen/Gruen.cpp
--- isis/src/base/objs/Gruen/Gruen.cpp	2018-04-06 15:12:15.056519393 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/Gruen/Gruen.cpp	2018-04-03 13:26:46.973145423 +0200
@@ -26,7 +26,7 @@
 #include "DbProfile.h"
 #include "Pvl.h"
 
-#include "tnt/tnt_array1d_utils.h"
+#include "tnt_array1d_utils.h"
 
 #include <cmath>
 #include <iostream>
diff -Naur isis/src/base/objs/Gruen/Gruen.h ../isis3_cmake_working/isis/src/base/objs/Gruen/Gruen.h
--- isis/src/base/objs/Gruen/Gruen.h	2018-04-06 15:12:15.056519393 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/Gruen/Gruen.h	2018-04-03 13:26:46.975145436 +0200
@@ -25,8 +25,8 @@
 
 
 #include "AutoReg.h"
-#include "tnt/tnt_array2d.h"
-#include "tnt/tnt_array1d.h"
+#include "tnt_array2d.h"
+#include "tnt_array1d.h"
 #include "Affine.h"
 #include "DbProfile.h"
 #include "GruenTypes.h"
diff -Naur isis/src/base/objs/Gruen/GruenTypes.h ../isis3_cmake_working/isis/src/base/objs/Gruen/GruenTypes.h
--- isis/src/base/objs/Gruen/GruenTypes.h	2018-04-06 15:12:15.056519393 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/Gruen/GruenTypes.h	2018-04-03 13:26:46.974145430 +0200
@@ -28,9 +28,9 @@
 #include <sstream>
 #include <iomanip>
 
-#include <tnt/tnt_array1d.h>
-#include <tnt/tnt_array2d.h>
-#include <tnt/tnt_array2d_utils.h>
+#include <tnt_array1d.h>
+#include <tnt_array2d.h>
+#include <tnt_array2d_utils.h>
 
 #include "Affine.h"
 #include "Chip.h"
diff -Naur isis/src/base/objs/LeastSquares/LeastSquares.cpp ../isis3_cmake_working/isis/src/base/objs/LeastSquares/LeastSquares.cpp
--- isis/src/base/objs/LeastSquares/LeastSquares.cpp	2018-04-06 15:12:15.071519486 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/LeastSquares/LeastSquares.cpp	2018-04-03 13:26:46.757144055 +0200
@@ -20,8 +20,8 @@
  *   http://isis.astrogeology.usgs.gov, and the USGS privacy and disclaimers on
  *   http://www.usgs.gov/privacy.html.
  */
-#include "jama/jama_svd.h"
-#include "jama/jama_qr.h"
+#include "jama_svd.h"
+#include "jama_qr.h"
 
 #include "LeastSquares.h"
 #include "IException.h"
diff -Naur isis/src/base/objs/LeastSquares/LeastSquares.h ../isis3_cmake_working/isis/src/base/objs/LeastSquares/LeastSquares.h
--- isis/src/base/objs/LeastSquares/LeastSquares.h	2018-04-06 15:12:15.071519486 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/LeastSquares/LeastSquares.h	2018-04-03 13:26:46.757144055 +0200
@@ -24,7 +24,7 @@
  */
 #include <vector>
 
-#include "tnt/tnt_array2d.h"
+#include "tnt_array2d.h"
 
 #include "BasisFunction.h"
 
diff -Naur isis/src/base/objs/Matrix/Matrix.cpp ../isis3_cmake_working/isis/src/base/objs/Matrix/Matrix.cpp
--- isis/src/base/objs/Matrix/Matrix.cpp	2018-04-06 15:12:15.074519505 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/Matrix/Matrix.cpp	2018-04-03 13:26:46.830144518 +0200
@@ -23,9 +23,9 @@
 #include <vector>
 #include <iostream>
 #include <string>
-#include "jama/jama_svd.h"
-#include "jama/jama_eig.h"
-#include "jama/jama_lu.h"
+#include "jama_svd.h"
+#include "jama_eig.h"
+#include "jama_lu.h"
 
 #include "Constants.h"
 #include "IException.h"
diff -Naur isis/src/base/objs/Matrix/Matrix.h ../isis3_cmake_working/isis/src/base/objs/Matrix/Matrix.h
--- isis/src/base/objs/Matrix/Matrix.h	2018-04-06 15:12:15.074519505 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/Matrix/Matrix.h	2018-04-03 13:26:46.830144518 +0200
@@ -23,7 +23,7 @@
  */
 
 #include <vector>
-#include "tnt/tnt_array2d.h"
+#include "tnt_array2d.h"
 
 namespace Isis {
   /**
diff -Naur isis/src/base/objs/NaifDskApi/NaifDskApi.cpp ../isis3_cmake_working/isis/src/base/objs/NaifDskApi/NaifDskApi.cpp
--- isis/src/base/objs/NaifDskApi/NaifDskApi.cpp	2018-04-06 15:12:15.076519517 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/NaifDskApi/NaifDskApi.cpp	2018-04-03 13:26:47.294147457 +0200
@@ -24,10 +24,10 @@
 #include "NaifDskApi.h"
 
 // Specs for convenient NAIF vectors and matrices
-#include <tnt/tnt_array1d.h>
-#include <tnt/tnt_array1d_utils.h>
-#include <tnt/tnt_array2d.h>
-#include <tnt/tnt_array2d_utils.h>
+#include <tnt_array1d.h>
+#include <tnt_array1d_utils.h>
+#include <tnt_array2d.h>
+#include <tnt_array2d_utils.h>
 
 #include "IString.h"
 
diff -Naur isis/src/base/objs/NaifDskApi/NaifDskApi.h ../isis3_cmake_working/isis/src/base/objs/NaifDskApi/NaifDskApi.h
--- isis/src/base/objs/NaifDskApi/NaifDskApi.h	2018-04-06 15:12:15.076519517 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/NaifDskApi/NaifDskApi.h	2018-04-03 13:26:47.295147463 +0200
@@ -31,10 +31,10 @@
 #include <SpiceZfc.h>
 
 // Specs for convenient NAIF vectors and matrices
-#include <tnt/tnt_array1d.h>
-#include <tnt/tnt_array1d_utils.h>
-#include <tnt/tnt_array2d.h>
-#include <tnt/tnt_array2d_utils.h>
+#include <tnt_array1d.h>
+#include <tnt_array1d_utils.h>
+#include <tnt_array2d.h>
+#include <tnt_array2d_utils.h>
 
 namespace Isis {
 
diff -Naur isis/src/base/objs/PrincipalComponentAnalysis/PrincipalComponentAnalysis.cpp ../isis3_cmake_working/isis/src/base/objs/PrincipalComponentAnalysis/PrincipalComponentAnalysis.cpp
--- isis/src/base/objs/PrincipalComponentAnalysis/PrincipalComponentAnalysis.cpp	2018-04-06 15:12:15.090519604 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/PrincipalComponentAnalysis/PrincipalComponentAnalysis.cpp	2018-04-03 13:26:46.919145081 +0200
@@ -23,8 +23,8 @@
 #include <QString>
 
 #include "PrincipalComponentAnalysis.h"
-#include "jama/jama_eig.h"
-#include "jama/jama_lu.h"
+#include "jama_eig.h"
+#include "jama_lu.h"
 
 using namespace std;
 
diff -Naur isis/src/base/objs/PrincipalComponentAnalysis/PrincipalComponentAnalysis.h ../isis3_cmake_working/isis/src/base/objs/PrincipalComponentAnalysis/PrincipalComponentAnalysis.h
--- isis/src/base/objs/PrincipalComponentAnalysis/PrincipalComponentAnalysis.h	2018-04-06 15:12:15.090519604 +0200
+++ ./isis3_cmake_working/isis/src/base/objs/PrincipalComponentAnalysis/PrincipalComponentAnalysis.h	2018-04-03 13:26:46.919145081 +0200
@@ -23,7 +23,7 @@
  */
 
 #include <vector>
-#include "tnt/tnt_array2d.h"
+#include "tnt_array2d.h"
 #include "MultivariateStatistics.h"
 #include "IException.h"
 #include "Constants.h"
diff -Naur isis/src/control/apps/cnetthinner/CnetManager.cpp ../isis3_cmake_working/isis/src/control/apps/cnetthinner/CnetManager.cpp
--- isis/src/control/apps/cnetthinner/CnetManager.cpp	2018-04-06 15:12:15.171520106 +0200
+++ ./isis3_cmake_working/isis/src/control/apps/cnetthinner/CnetManager.cpp	2018-04-03 13:26:46.234140743 +0200
@@ -40,8 +40,8 @@
 #include "ControlMeasure.h"
 #include "ControlMeasureLogData.h"
 
-#include "tnt/tnt_array2d.h"
-#include "tnt/tnt_array2d_utils.h"
+#include "tnt_array2d.h"
+#include "tnt_array2d_utils.h"
 
 ///#define DEBUG 1
 
diff -Naur isis/src/control/apps/cnetthinner/CnetManager.h ../isis3_cmake_working/isis/src/control/apps/cnetthinner/CnetManager.h
--- isis/src/control/apps/cnetthinner/CnetManager.h	2018-04-06 15:12:15.171520106 +0200
+++ ./isis3_cmake_working/isis/src/control/apps/cnetthinner/CnetManager.h	2018-04-03 13:26:46.236140756 +0200
@@ -45,8 +45,8 @@
 #include "ControlMeasureLogData.h"
 
 
-#include "tnt/tnt_array2d.h"
-#include "tnt/tnt_array2d_utils.h"
+#include "tnt_array2d.h"
+#include "tnt_array2d_utils.h"
 
 namespace Isis {
 
diff -Naur isis/src/control/apps/cnetthinner/CnetSuppression.cpp ../isis3_cmake_working/isis/src/control/apps/cnetthinner/CnetSuppression.cpp
--- isis/src/control/apps/cnetthinner/CnetSuppression.cpp	2018-04-06 15:12:15.171520106 +0200
+++ ./isis3_cmake_working/isis/src/control/apps/cnetthinner/CnetSuppression.cpp	2018-04-03 13:26:46.235140750 +0200
@@ -40,8 +40,8 @@
 #include "IException.h"
 #include "Progress.h"
 
-#include "tnt/tnt_array2d.h"
-#include "tnt/tnt_array2d_utils.h"
+#include "tnt_array2d.h"
+#include "tnt_array2d_utils.h"
 
 #define EARLY_TERMINATION  true
 #define RADIUS_CELLS        100
diff -Naur isis/src/control/apps/cnetthinner/CnetSuppression.h ../isis3_cmake_working/isis/src/control/apps/cnetthinner/CnetSuppression.h
--- isis/src/control/apps/cnetthinner/CnetSuppression.h	2018-04-06 15:12:15.171520106 +0200
+++ ./isis3_cmake_working/isis/src/control/apps/cnetthinner/CnetSuppression.h	2018-04-03 13:26:46.234140743 +0200
@@ -38,8 +38,8 @@
 #include "CnetManager.h"
 #include "Progress.h"
 
-#include "tnt/tnt_array2d.h"
-#include "tnt/tnt_array2d_utils.h"
+#include "tnt_array2d.h"
+#include "tnt_array2d_utils.h"
 
 namespace Isis {
 
diff -Naur isis/src/dawn/objs/DawnVirCamera/DawnVirCamera.cpp ../isis3_cmake_working/isis/src/dawn/objs/DawnVirCamera/DawnVirCamera.cpp
--- isis/src/dawn/objs/DawnVirCamera/DawnVirCamera.cpp	2018-04-06 15:12:15.206520323 +0200
+++ ./isis3_cmake_working/isis/src/dawn/objs/DawnVirCamera/DawnVirCamera.cpp	2018-04-03 13:26:46.354141503 +0200
@@ -9,7 +9,7 @@
 #include <QRegExp>
 #include <QString>
 
-#include <tnt/tnt_array2d_utils.h>
+#include <tnt_array2d_utils.h>
 
 #include "Camera.h"
 #include "CameraFocalPlaneMap.h"
diff -Naur isis/src/dawn/objs/DawnVirCamera/DawnVirCamera.h ../isis3_cmake_working/isis/src/dawn/objs/DawnVirCamera/DawnVirCamera.h
--- isis/src/dawn/objs/DawnVirCamera/DawnVirCamera.h	2018-04-06 15:12:15.206520323 +0200
+++ ./isis3_cmake_working/isis/src/dawn/objs/DawnVirCamera/DawnVirCamera.h	2018-04-03 13:26:46.355141510 +0200
@@ -5,7 +5,7 @@
 
 #include <QString>
 
-#include <tnt/tnt_array2d.h>
+#include <tnt_array2d.h>
 
 #include "VariableLineScanCameraDetectorMap.h"
 
diff -Naur isis/src/rosetta/objs/RosettaVirtisCamera/RosettaVirtisCamera.cpp ../isis3_cmake_working/isis/src/rosetta/objs/RosettaVirtisCamera/RosettaVirtisCamera.cpp
--- isis/src/rosetta/objs/RosettaVirtisCamera/RosettaVirtisCamera.cpp	2018-04-06 15:12:15.493522102 +0200
+++ ./isis3_cmake_working/isis/src/rosetta/objs/RosettaVirtisCamera/RosettaVirtisCamera.cpp	2018-04-03 13:26:46.321141294 +0200
@@ -9,7 +9,7 @@
 #include <QRegExp>
 #include <QString>
 
-#include <tnt/tnt_array2d_utils.h>
+#include <tnt_array2d_utils.h>
 
 #include "Camera.h"
 #include "CameraFocalPlaneMap.h"
diff -Naur isis/src/rosetta/objs/RosettaVirtisCamera/RosettaVirtisCamera.h ../isis3_cmake_working/isis/src/rosetta/objs/RosettaVirtisCamera/RosettaVirtisCamera.h
--- isis/src/rosetta/objs/RosettaVirtisCamera/RosettaVirtisCamera.h	2018-04-06 15:12:15.493522102 +0200
+++ ./isis3_cmake_working/isis/src/rosetta/objs/RosettaVirtisCamera/RosettaVirtisCamera.h	2018-04-03 13:26:46.321141294 +0200
@@ -5,7 +5,7 @@
 
 #include <QString>
 
-#include <tnt/tnt_array2d.h>
+#include <tnt_array2d.h>
 
 #include "VariableLineScanCameraDetectorMap.h"
 
