--- ./isis/cmake/FindCholmod.cmake	2018-04-06 15:12:14.690517125 +0200
+++ ./isis3_cmake_working/isis/cmake/FindCholmod.cmake	2018-03-29 14:46:30.658410013 +0200
@@ -28,7 +28,7 @@
 # if(APPLE)
   find_library(FORTRAN_LIBRARY      NAMES gfortran
     NAMES gfortran
-    PATHS /opt/usgs/v007/ports/lib/gcc5/
+    PATHS /opt/usgs/v007/ports/lib/gcc5/ /usr/lib64/gcc/x86_64-pc-linux-gnu/6.4.0/ /usr/lib64/gcc/x86_64-pc-linux-gnu/7.3.0/
   )
 
   find_library(BLAS_LIBRARY NAMES blas)
